draft-ietf-quic-recovery.md

```diff
@@ -819,6 +819,16 @@ kGranularity:
 kInitialRtt:
 : The RTT used before an RTT sample is taken. The RECOMMENDED value is 100ms.
 
+kPacketNumberSpace:
+: An enum to enumerate the three packet number spaces.
+~~~
+  enum kPacketNumberSpace {
+    Initial,
+    Handshake,
+    1-RTT,
```

Comment:
I think the name for the PN space that governs 0- and 1-RTT packets is “application data”.

Reply:
Done.